content/doc/developer/tutorial/prepare.adoc

```diff
@@ -40,9 +40,11 @@ On macOS, the link:https://brew.sh/[Homebrew] package manager offers Maven packa
 Make sure a recent version of Maven 3, ideally 3.8.3 or newer, is provided if you decide to go this route.
 // TIMEBASED
 
-Extract Maven, and take note of its location. Then, add the the full path of the `bin/` subdirectory extracted (for example, `~/Applications/apache-maven/bin/mvn` or `C:\Program files\Maven\bin\mvn`) to the `PATH` variable in your OS. (This will let you invoke Maven using `mvn`).
+Next you will need to extract Maven, and take note of its location. When you extract the Maven files, make sure you extract them directly into the target repository (e.g. extract straight into C:\Program Files (x86)\Maven ) - do not extract the files to a different location then copy the files.
 
-The rest of the tutorial assumes that Maven is on your `PATH`, but you can also just type the full path.
+Then, add the the full path of the `bin/` subdirectory extracted (for example, `~/Applications/apache-maven/bin/mvn` or `C:\Program files\Maven\bin\mvn`) to the `PATH` variable in your OS. (This will let you invoke Maven using `mvn`).
```

Comment:
Let's remove the duplicate 'the' and make the parenthetical sentence into a sentence without parentheses.
```suggestion
Then, add the full path of the `bin/` subdirectory extracted (for example, `~/Applications/apache-maven/bin/mvn` or `C:\Program files\Maven\bin\mvn`) to the `PATH` variable in your OS.
This will let you invoke Maven using `mvn`.
```